pkg/v1/providers/infrastructure-vsphere/v1.3.1/yttcc/overlay.yaml

```diff
@@ -42,7 +42,7 @@ spec:
     #! VVV TODO(vui) compute
     #! version: #@ data.values.KUBERNETES_VERSION
     #! hack to match up tkr
-    version: #@ "{}-tkg.1-zshippable".format(data.values.KUBERNETES_VERSION)
+    version: #@ "{}-tkg.2-zshippable".format(data.values.KUBERNETES_VERSION)
```

Comment:
TKR resolver can take care of this from just Kubernetes version. It will get replaced with the one from the resolved TKR (if a matching one exists, of course).

Reply:
OK, I will close this issue so, and rely on TKr resolution, thanks.